pkg/csi/cinder/controllerserver.go

```diff
@@ -160,6 +164,10 @@ func (cs *controllerServer) ControllerPublishVolume(ctx context.Context, req *cs
 
 	err = cs.Cloud.WaitDiskAttached(instanceID, volumeID)
 	if err != nil {
+		if cpoerrors.IsNotFound(err) {
```

Comment:
This might never return nofound error
https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/openstack/openstack_volumes.go#L164

Reply:
ok, let's remove this check ,thanks for pointing out..